sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/util/DateTimeUtils.scala

```diff
@@ -62,6 +64,11 @@ object DateTimeUtils {
 
   @transient lazy val defaultTimeZone = TimeZone.getDefault
 
+  // These are ISO 8601 compliant `FastDateFormat`s. These are thread-safe and compatible
+  // with `SimpleDateFormat`.
+  lazy val iso8601TimeFormat = FastDateFormat.getInstance("yyyy-MM-dd'T'HH:mm:ss.SSSZZ")
```

Comment:
I thought this might be used or not. As far as I know, making and parsing the date pattern is a bit expensive. So, I just wanted to avoid to initiate this variable when it is not used. Does this make sense?